x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts

```diff
@@ -26,6 +26,7 @@ const configSchema = schema.object({
       'Host Flyout Filter Added', // Worst-case scenario once per second - AT RISK,
     ],
   }),
+  pageVarsDebounceTimeMs: schema.duration({ defaultValue: '500ms' }),
```

Comment:
I find this setting a bit confusing. I would expect `pageVarsDebounceTimeMs` to be of type number and allow customers to specify the duration in milliseconds without specifying the unit. 

I would either remove the "Ms" bit from the name or enforce type number.

Reply:
Good point! In https://github.com/elastic/kibana/pull/171450/commits/45ba07bf52048f3aa39bbf1ff0634c2ffc4a786c I've removed the `Ms` in those places where we are dealing with durations :)